criu/config.c

```diff
@@ -1036,6 +1036,8 @@ int parse_options(int argc, char **argv, bool *usage_error, bool *has_exec_cmd,
 				opts.network_lock_method = NETWORK_LOCK_IPTABLES;
 			} else if (!strcmp("nftables", optarg)) {
 				opts.network_lock_method = NETWORK_LOCK_NFTABLES;
+			} else if (!strcmp("none", optarg)) {
```

Comment:
Would it be possible to update the man page as well?
https://github.com/checkpoint-restore/criu/blob/c75c017e/Documentation/criu.txt#L453